tasks/install_psmdb_full.yml

```diff
@@ -41,4 +44,7 @@
         - percona-server-mongodb-shell
         - percona-server-mongodb-tools
         - percona-server-mongodb-debugsource
```

Comment:
`percona-server-mongodb-server-debuginfo` should probably be added also here.

Reply:
There is no percona-server-mongodb-server-debuginfo package, there is  percona-server-mongodb-debugsource instead.